Dockerfile

```diff
@@ -1,20 +1,26 @@
-FROM rilldata/duckdb
+FROM rilldata/duckdb:0.3.3
 
 WORKDIR /app
 
 COPY package.json package-lock.json \
      tsconfig.json tsconfig.node.json tsconfig.build.json \
      svelte.config.js tailwind.config.cjs postcss.config.cjs .babelrc ./
+
+COPY build-tools build-tools/
 COPY src src/
+
```

Comment:
btw, can we make `npm install` not depend on the src, so that we dont have to install the dependencies everytime we build the docker image.